chapter2/tests/t20/terraform/base/config.tf

```diff
@@ -0,0 +1,23 @@
+terraform {
+  required_version = ">= 1.4.0"
+
+  required_providers {
+    aws = "~> 4.0"
+
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.5.1"
+    }
+    null = "~> 3.2.1"
+  }
+}
```

Comment:
when you create new tf code always
- check versions of modules and use latest
- check versions of providers and use latest

that will simplify efforts in future maintenance/updates

Reply:
> when you create new tf code always

@dmitry-mightydevops 
Done